### PR TITLE
Cleanup effects

### PR DESCRIPTION
**Pull request by mkrus**
Hack in no longer needed with change in qt5.12. Actually wonder if it should be conditionally compiled in if qt ver is < 5.12.2?

Also includes some clang-tidy formatting changes.

### CODE REVIEW

File: src/core/metallicroughnesseffect.cpp [hunk 1]
```diff
@@ -205,14 +205,14 @@ MetallicRoughnessEffect::MetallicRoughnessEffect(Qt3DCore::QNode *parent)
...
 {
-    const auto enabledLayers = QStringList{ QStringLiteral("noBaseColorMap"),
-                                            QStringLiteral("noMetalRoughMap"),
-                                            QStringLiteral("noAmbientOcclusionMap"),
-                                            QStringLiteral("noEmissiveMap"),
-                                            QStringLiteral("noNormalMap"),
-                                            QStringLiteral("noHasColorAttr"),
-                                            QStringLiteral("noDoubleSided"),
-                                            QStringLiteral("noHasAlphaCutoff") };
+    const auto enabledLayers = QStringList { QStringLiteral("noBaseColorMap"),
```
Comment (er-vin) [inline src/core/metallicroughnesseffect.cpp]: Hm... this was done this way by clang-format. Are you sure about that format change?

File: src/core/metallicroughnesseffect.cpp [hunk 2]
```diff
@@ -351,15 +351,6 @@ MetallicRoughnessEffect::MetallicRoughnessEffect(Qt3DCore::QNode *parent)
...
 
-    // Note that even though those parameters are not exposed in the API,
-    // they need to be kept around for now due to a bug in some drivers/GPUs
-    // (at least Intel) which cause issues with unbound textures even if you
-    // don't try to sample from them.
-    // Can probably go away once we generate the shaders and deal in this
-    // case in a better way.
-    addParameter(new QParameter(QStringLiteral("envLight.irradiance"), new QTexture2D));
-    addParameter(new QParameter(QStringLiteral("envLight.specular"), new QTexture2D));
-
```
Comment (er-vin) [inline src/core/metallicroughnesseffect.cpp]: I'm not sure we can remove it now... I mean due to ABI compatibility it could run on any 5.12.x version and only 5.12.2 will carry the fix. I'd be tempted to make that conditional based on 5.13 being detected at build time.

Comment (mkrus) [reply src/core/metallicroughnesseffect.cpp]: done

File: tests/manual/forwardrenderer_scene/main.qml [hunk 3]
```diff
@@ -51,8 +51,7 @@ Entity {
         }
     ]
 
-    function enablePostProcessingEffect(effect, enable)
-    {
+    function enablePostProcessingEffect(effect, enable) {
```
Comment (jjcasmar) [inline tests/manual/forwardrenderer_scene/main.qml]: Unrelated?

Comment (jjcasmar) [reply tests/manual/forwardrenderer_scene/main.qml]: Dont think so
